bg_atlasgen/wrapup.py

```diff
@@ -158,15 +158,15 @@ def wrapup_atlas_from_data(
     for mesh_id, meshfile in meshes_dict.items():
         mesh = mio.read(meshfile)
 
+        if scale_meshes:
+            # Scale the mesh to be in microns, BEFORE transforming:
```

Comment:
```suggestion
            # Scale the mesh to the desired resolution, BEFORE transforming:
```

Nit-picky, but isn't this a general (re)-scale based on the resolution given' so doesn't necessarily scale the mesh to be in microns?

Unless there's some assumption on the `resolution` variable in the unchanged code I'm missing.